Composer/packages/server/src/models/bot/__tests__/orchestratorBuilder.test.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { FileInfo } from '@bfc/types';
+import { Utility } from '@microsoft/bf-orchestrator';
+import rimraf from 'rimraf';
+
+import * as path from 'path';
+import { IFileStorage } from '../../storage/interface';
+import { Builder } from '../builder';
+import mockLUInput from '../__mocks__/mockLUInput.json';
+
+const nlrId: string = 'pretrained.20200924.microsoft.dte.00.03.en.onnx';
+const nlrPath: string = path.resolve('./orchestrator_ut_model/');
+const downloadModelTimeoutMs = 90000;
+
+describe('Orchestrator Tests', () => {
+  beforeAll(async () => {
+    // disable Orchestrator Lib console logging across the board - interferes with Jest
+    Utility.toPrintDebuggingLogToConsole = false;
+
+    const progressStatusStub = jest.fn();
+    const successStatusStub = jest.fn();
+
+    let builder = new Builder('', {} as IFileStorage, 'en-us');
+
+    // download the UT NLR model once before all tests are run - build tests don't work without a valid model
+    await builder.runOrchestratorNlrGet(nlrPath, nlrId, progressStatusStub, successStatusStub);
```

Comment:
I've added this UT to the jest ignore list for now, since a 180MB+ binary file download is required to run this UT.  I'm aware this is a code-smell, any suggestions appreciated. 

Orchestrator team have mentioned making a small model available for testing purposes in the future.